torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -61,5 +61,15 @@ struct TORCH_API SoftmaxOptions {
   TORCH_ARG(int64_t, dim);
 };
 
+// ============================================================================
+
+/// Options for the Softmin functional and module.
+struct TORCH_API SoftminOptions {
+  SoftminOptions(int64_t dim);
+
+  // Dimension along which Softmin will be computed.
```

Comment:
nit: we should change `//` to `///` so that the comment will show up in the C++ API docs (my apologies for missing it for the `Softmax` module 😞)

Reply:
@yf225 

If you don't mind, I'll also fix `Softmax` module in this PR.
What do you think about this idea?

Reply:
Yes that would be awesome, thanks so much for your help @nuka137!

Reply:
Fixed them.